lib/stat_tracker.rb

```diff
@@ -0,0 +1,52 @@
+require 'csv'
+# require_relative '../lib/game'
+# require_relative '../lib/team'
+# require_relative '../lib/game_team'
+require_relative './game'
+require_relative './team'
+require_relative './game_team'
+#
+
+class StatTracker
+
+  def self.from_csv(file_paths)
+    game_teams_path = file_paths[:game_teams]
+    games_path = file_paths[:games]
+    teams_path = file_paths[:teams]
+
+    self.new(game_teams_path, games_path, teams_path)
+
+  end
+
+  attr_reader :game_teams_path,
+              :games_path,
+              :teams_path,
```

Comment:
Aurora said not to run the .from_csv every time we want to access the collection. This runs 3 collection methods to create the collection instance variables

Reply:
thank you for solving this problem. I hadn't yet figured it out!

Reply:
wow! this is great it will definitely help us out in the long run!